.github/workflows/rollback.yml

```
@@ -0,0 +1,386 @@
+        id: target
+        run: |
+          if [ -n "${{ needs.validate.outputs.target-deployment }}" ]; then
+            target="${{ needs.validate.outputs.target-deployment }}"
+            echo "Using specified target deployment: $target"
+          else
+            echo "🔍 Finding previous deployment..."
+            # Get the second most recent production deployment
+            target=$(vercel ls --token=${{ secrets.VERCEL_TOKEN }} --scope=${{ secrets.VERCEL_ORG_ID }} ${{ secrets.VERCEL_PROJECT_ID }} --meta production=true -t 2 | tail -1)
+            echo "Using previous deployment: $target"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)